impeller/entity/contents/runtime_effect_contents.cc

```diff
@@ -182,7 +182,7 @@ bool RuntimeEffectContents::Render(const ContentContext& renderer,
 
         ShaderUniformSlot uniform_slot;
         uniform_slot.name = uniform.name.c_str();
-        uniform_slot.ext_res_0 = buffer_index;
+        uniform_slot.ext_res_0 = uniform.location;
```

Comment:
In practice buffer_index and uniform.location are the same now but I believe this is more correct.